src/matrix/room/BaseRoom.js

```diff
@@ -378,8 +381,44 @@ export class BaseRoom extends EventEmitter {
         return this._summary.data.membership;
     }
 
-    get powerLevels() {
-        return this._timeline.powerLevels;
+    async loadPowerLevels() {
+        const timelineReader = new TimelineReader({
+            roomId: this._roomId,
+            storage: this._storage,
+            fragmentIdComparer: this._fragmentIdComparer
+        });
+        const txn = await this._storage.readTxn(
+            timelineReader.readTxnStores.concat(this._storage.storeNames.roomMembers, this._storage.storeNames.roomState)
+        );
+        const powerLevelsState = await txn.roomState.get(this._roomId, "m.room.power_levels", "");
+        if (powerLevelsState) {
+            return new PowerLevels({
+                powerLevelEvent: powerLevelsState.event,
+                ownUserId: this._user.id,
+                membership: this.membership
+            });
+        }
+        const createState = await txn.roomState.get(this._roomId, "m.room.create", "");
+        if (createState) {
+            return new PowerLevels({
+                createEvent: createState.event,
+                ownUserId: this._user.id,
+                membership: this.membership
+            });
+        } else {
+            const membership = this.membership;
+            return new PowerLevels({ownUserId: this._user.id, membership});
+        }
+    }
+
+    async observePowerLevels() {
+        let observable = this._powerLevels;
```

Comment:
as the function currently is, if a second call is made before `load...` finishes for the first call, it will load the power levels twice. I think you will need to cache the promise from `load...` and store it on `this` and await that if it is set.